cmd/gokey/main.go

```diff
@@ -79,20 +83,43 @@ func genKey(seed []byte, w io.Writer) {
 	}
 }
 
-// TODO: parametrize size
-// generates raw 32 bytes
 func genRaw(seed []byte, w io.Writer) {
+	if isInvalidLength(length) {
```

Comment:
same - let's move validation to the `main` function

Reply:
5d166b2